tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py

```diff
@@ -64,9 +57,9 @@ def test_add_capacity(
 
         # All OCS pods are in running state:
         # ToDo https://github.com/red-hat-storage/ocs-ci/issues/2361
-        assert (
-            pod_helpers.check_pods_in_running_state()
-        ), "Entry criteria FAILED: one or more OCS pods are not in running state"
+        # assert (
+        #     pod_helpers.check_pods_in_running_state()
+        # ), "Entry criteria FAILED: one or more OCS pods are not in running state"
```

Comment:
Any specific reason for commenting this out? Is it temporary or permanent?
Please note that this automated test is derived from the Polarion test where this entry criteria is mentioned.